core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -131,7 +131,7 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       for {
         // we exclude workflows submitted by users that have exceeded the max workflows per user
         excludedUsers <- dataAccess.workflowQuery.listSubmittersWithMoreWorkflowsThan(maxActiveWorkflowsPerUser, WorkflowStatuses.runningStatuses)
-        workflowRecs <- dataAccess.workflowQuery.findQueuedWorkflows(excludedUsers.map { case (submitter, count) => submitter }).take(batchSize).result
+        workflowRecs <- dataAccess.workflowQuery.findQueuedWorkflows(excludedUsers.map { case (submitter, count) => submitter }, Seq(SubmissionStatuses.Aborting)).take(batchSize).result
```

Comment:
Are there other Submission statuses we should exclude, e.g. `Aborted` or `Failed` ?